app/src/main/java/com/example/mydigitalcloset/SettingsPage.java

```diff
@@ -6,28 +6,45 @@
 import android.os.Bundle;
 import android.util.Log;
 import android.view.View;
-import android.widget.ArrayAdapter;
+
 import android.widget.Button;
-import android.widget.ListView;
-import android.widget.TextView;
+
 import android.widget.Toast;
 
-import com.example.mydigitalcloset.closetPage.AllSavedOufitsPage;
 import com.example.mydigitalcloset.closetPage.AllSavedOutfits;
-import com.example.mydigitalcloset.databinding.ActivityAboutusPageBinding;
-import com.example.mydigitalcloset.databinding.ActivityOutfitCreationBinding;
 import com.example.mydigitalcloset.databinding.ActivitySettingsPageBinding;
 
+
+import com.google.firebase.auth.FirebaseAuth;
+
 public class SettingsPage extends AppCompatActivity {
 
     ActivitySettingsPageBinding binding;
 
+    private FirebaseAuth mAuth;
+
     @Override protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
         // Inflate the layout
         binding = ActivitySettingsPageBinding.inflate(getLayoutInflater());
         setContentView(binding.getRoot());
 
+        mAuth = FirebaseAuth.getInstance();
+        Button btnLogOut = findViewById(R.id.btnLogOut);
+
+        btnLogOut.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                mAuth.signOut();
+                Intent intent = new Intent(SettingsPage.this, Login.class);
```

Comment:
double check to see if you can log back in after this, can't tell if it will actually log user out

Reply:
Yes, once the user logs out you can log back into the app.